tests/system_tests_tcp_adaptor.py

```diff
@@ -1045,11 +1090,16 @@ def run_ncat(self,
                  timeout=10,
                  data=b'abcd',
                  use_ssl=False,
-                 use_client_cert=False):
+                 use_client_cert=False,
```

Comment:
Just my opinion - you don't have to do it:

Why not just pass the actual client cert and key file as parameters rather than passing a bunch of flags and having run_ncat figure out what the certs are based on those flags?

IOW:

def run_ncat(... certfile=None, keyfile=None):...